### DISCUSSION TITLE
Unable to program ESP32 with latest board rev.

### DISCUSSION THREAD

**Discussion by KiloWattWizard** [revk/xESP32-Faikout#514] [General]
I'm pretty new to using the ESP32 outside of dev boards, so I might be missing something obvious here.

I have ordered a latest-revision board from JLCPCB and I'm trying to program the firmware. I can't seem to get the ESP32 to respond to programming attempts. I have a USB breakout soldered directly to the pads for USB, and I have verified that I've got +5 and 3.3V on the board.

The Espressif documentation states that you need to pull GPIO0 down and hit reset to get into DFU mode, but these pins are not broken out on the board.

What am I missing?

**Comment by revk** [comment]
USB just works, no need to mess with GPIO0, only connection is to the programming pads, with D-, D+, GND, and 5V.

**Reply by KiloWattWizard** [reply]
Got it thanks.
